list index out of range for pypi/rst2pdf/0.93

```python
{"node_args": {"recursive_limit": 0, "ecosystem": "pypi", "version": "0.93", "name": "rst2pdf", "force_graph_sync": false, "force": true}, "task_name": "InitAnalysisFlow", "record_id": null, "error_traceback": "  File \"/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py\", line 114, in run\n    result = task.run(node_args)\n  File \"/usr/lib/python3.4/site-packages/f8a_worker/base.py\", line 38, in run\n    result = self.execute(node_args)\n  File \"/usr/lib/python3.4/site-packages/f8a_worker/workers/init_analysis_flow.py\", line 47, in execute\n    target_dir=cache_path\n  File \"/usr/lib/python3.4/site-packages/f8a_worker/process.py\", line 271, in fetch_artifact\n    file_url = release_files[0]['url']\n", "storage_task_name": "InitAnalysisFlow", "flow_name": "bayesianFlow", "error_type": "<class 'IndexError'>", "event": "STORAGE_STORE_ERROR", "error_value": "list index out of range", "storage_name": "BayesianPostgres", "task_id": "0e72f7e2-9554-4c81-b7fe-1fec53aca450"}
2017-07-17 08:43:37,111 [ERROR] celery.app.trace: Task selinon.SelinonTaskEnvelope[0e72f7e2-9554-4c81-b7fe-1fec53aca450] raised unexpected: IndexError('list index out of range',)
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 367, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/celery/app/trace.py", line 622, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 169, in run
    raise self.retry(max_retries=0, exc=exc)
  File "/usr/lib/python3.4/site-packages/celery/app/task.py", line 668, in retry
    raise_with_context(exc)
  File "/usr/lib/python3.4/site-packages/selinon/selinonTaskEnvelope.py", line 114, in run
    result = task.run(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/base.py", line 38, in run
    result = self.execute(node_args)
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/init_analysis_flow.py", line 47, in execute
    target_dir=cache_path
  File "/usr/lib/python3.4/site-packages/f8a_worker/process.py", line 271, in fetch_artifact
    file_url = release_files[0]['url']
IndexError: list index out of range
```

--- a/f8a_worker/process.py
+++ b/f8a_worker/process.py
@@ -258,7 +258,9 @@ class IndianaJones(object):
             res.raise_for_status()
             if not version:
                 version = res.json()['info']['version']
-            release_files = res.json()['releases'][version]
+            release_files = res.json().get('releases', {}).get(version, [])
+            if not release_files:
+                raise RuntimeError("No release files for version %s" % version)
 
             # sort releases by order in which we'd like to download:
             #  1) sdist

